sway-lib-std/src/storage/storage_map.sw

```diff
@@ -125,4 +133,69 @@ where
         let key = sha256((key, self.slot));
         clear::<V>(key, 0)
     }
+
+    /// Inserts a key-value pair into the map, if a value does not already exist for the key.
+    ///
+    /// # Arguments
+    ///
+    /// * `key`: [K] - The key to which the value is paired.
+    /// * `value`: [V] - The value to be stored.
+    ///
+    /// # Returns
+    ///
+    /// * [Result<V, StorageMapError<V>>] - `Result::Ok(value)` if the value was inserted, or `Result::Err(StorageMapError::OccupiedError(pre_existing_value))` if a value already existed for the key.
+    ///
+    /// # Number of Storage Accesses
+    ///
+    /// * Reads: `1`
+    /// * Writes: `1`
+    ///
+    /// # Examples
+    ///
+    /// ```sway
+    /// use std::storage::storage_map::StorageMapError;
```

Comment:
You have to import the error separately?

Reply:
Yeah since i use it in the assert